binding.c

```diff
@@ -6,6 +6,8 @@
 
 #if THREAD_STATS_APPLE
 #include "apple.h"
+#elif THREAD_STATS_LINUX
+#include "linux.c"
```

Comment:
```suggestion
#include "linux.h"
```

Only header files should be `#included`'ed so let's make it a header file.